### DISCUSSION TITLE
Inference Engine

### DISCUSSION THREAD

**Discussion by mssanamateen** [mssanamateen/Artificial-Intelligence-Lab#17] [Q&A] [answered]
What is the use of predicate logic in AI?

**Comment by YNandini60** [comment, marked as answer]
Predicate logic, also known as First Order Logic (FOL).
It is a mathematical language that helps computers make decisions and process information in AI.
It is mainly used as a foundational framework for representation of knowledge and reasoning.
It is used for break down sentences into smaller parts predicates and individuals.
It is uesd in Talks about variables
